Sources/Views/Collections/CollectionSection.swift

```diff
@@ -153,7 +153,7 @@ private func relatedItems(_ relatedItems: [Episode.Collection.Section.Related])
     .gridRow(
       attributes: [
         .class([
-          Class.grid.between(.desktop),
+          Class.padding([.mobile: [.topBottom: 2], .desktop: [.topBottom: 3]])
```

Comment:
`between` shouldn't be needed since there's only one column taking up the full width. and this padding is from me moving it out of a child and into this parent.